### PR TITLE
add database connect and change getUser function

### PR DESCRIPTION
**Pull request by hoseinsedaqat**

### CODE REVIEW

File: src/app/index.ts [hunk 1]
```diff
@@ -18,9 +20,20 @@ export const app = async (): Promise<void> => {
     res.status(500).json({ message: err.message });
   });
 
-  app.listen(port, () => {
-    console.log(`App is running on port ${port}`);
-  });
+  const uri: string = `mongodb://localhost:27017/wave_coding`;
```
Comment (zoheirkhonyagar) [inline src/app/index.ts:23]: @hsedaqat1378 please move this one to the `.env` file and update `.env.example` file

File: src/app/index.ts [hunk 2]
```diff
@@ -18,9 +20,20 @@ export const app = async (): Promise<void> => {
...
+
+  const options = { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: true, useCreateIndex: true };
+
+  mongoose.set('useFindAndModify', false);
+
+  mongoose.connect(uri, options).then(() =>
+    app.listen(port, () =>
+      console.log(`Server running on http://localhost:${port}`)
```
Comment (zoheirkhonyagar) [inline src/app/index.ts:31]: @hsedaqat1378 please read the connection URL string from `.env` file

File: src/app/user/user.service.ts [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+
 import User from './user.model';
+
 export class UserService {
+
   private model = User;
 
-  getUser(): string {
-    return '';
+  async getUser(req: Request, res: Response): Promise<void> {
```
Comment (zoheirkhonyagar) [inline src/app/user/user.service.ts:11]: @hsedaqat1378 also you can accept filters for the user so pass some input like `filters` and this filter could be some properties that user has and we can pass it to `where` condition on the query.

File: src/app/index.ts (hunk 1 above)
Comment (hoseinsedaqat) [reply src/app/index.ts:23]: I change this and thanks for tell me about this i know i'm write dirty code

File: src/app/index.ts (hunk 2 above)
Comment (hoseinsedaqat) [reply src/app/index.ts:31]: yeah change this and import url from env.ts file i think now that be good

File: src/app/user/user.service.ts (hunk 3 above)
Comment (hoseinsedaqat) [reply src/app/user/user.service.ts:11]: i not sure how change this but I'm try to better, Thanks a lot man

**Pull request closed without merging**